dashboard/config/routes.rb

```diff
@@ -7,6 +7,9 @@
   # Redirect studio.code.org/courses to code.org/students
   get "/courses", to: redirect(CDO.code_org_url("/students"))
 
+  # Redirect studio.code.org/mysection to most recent section section
```

Comment:
I don't think the path on this comment is right?

Reply:
You're right - that is dated.  I will update.